content/docs/user-guide/basic-concepts/workspace.md

```diff
@@ -6,3 +6,46 @@ tooltip: >-
   models, etc. Typically, it's also a Git repository. It will contain your DVC
   project.
 ---
+
+<!-- keywords: data science project architecture, machine learning project architecture, machine learning workflow, data science workflow, machine learning file system, data science file system, data science project structure, machine learning project structure, notebook version control -->
```

Comment:
Just a note to remove this comment later before merging (but for now it's useful to have the list during review 👍)

Reply:
Great job incorporating all of the terms BTW. We may have to drop a few but it's best to start having too many. Only "notebook version control" is missing but TBH I don't see how having that in workspace helps much — seems somewhat unrelated.

Reply:
😄 Actually the first two paragraphs are added yesterday. [Last week's workspace document](https://github.com/iterative/dvc.org/blob/ug-basic-concepts-feb-21/content/docs/user-guide/concepts/workspace.md) does not contain them. I can simply remove them. I also think they are not related.